app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,65 @@
+class InvitesController < ApplicationController
+
+    SUCCESS_MESSAGES = {
+        :create_success => "successfully invited user",
+        :accept_or_reject_success => "successfully submitted decision"
+    }
+
+    def outgoing
+        extract_arguments_from_params
+        render :json => { :outgoing => Invite.all_users_invited_by(@me).map(&:response_object) }
```

Comment:
Can this not just be a list? I guess you wanted to ensure it could be extended. I guess that's fine.